packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -16,13 +19,42 @@ export interface CardHeaderActionsObject {
   className?: string;
 }
 
+export interface CardHeaderSelectableActionsObject {
+  /** Determines the type of input to be used for a selectable card. */
+  variant?: 'single' | 'multiple';
+  /** Flag indicating that the actions have no offset */
+  hasNoOffset?: boolean;
+  /** Additional classes added to the selectable actions wrapper */
+  className?: string;
+  /** ID passed to the selectable or clickable input */
+  selectableActionId: string;
+  /** Adds an accessible label to the selectable or clickable input */
+  selectableActionAriaLabel?: string;
+  /** Adds an accessible label to the selectable or clickable input by passing in a
+   * space separated list of id's.
+   */
+  selectableActionAriaLabelledby?: string;
```

Comment:
May this should just be an array.

Reply:
We could, but a string type follows how other `aria-labelledby` props are set, and we'd have to turn that array into a space separated list of IDs internally since that's what aria-labelledby expects.

If anything it might be worth deciding whether we want to update these props across components to an array type

Reply:
I think this is fine then.